src/bento/components/bento-accordion/1.0/storybook/Basic.amp.js

```diff
@@ -1,86 +0,0 @@
-import {withAmp} from '@ampproject/storybook-addon';
```

Comment:
@rileyajones This was caused by duplication in `src/bento` and `extensions`.

It's simpler if we keep AMP Storybook files in `extensions/`, and Bento's in `src/bento`.

This means that:

- Files matching `storybook/*.amp.js` should remain in `extensions/`.
- Files matching `storybook/*.js`, without `.amp.js`, should be **moved**, not copied, to `src/bento`